app/forms/triage_form.rb

```diff
@@ -58,7 +61,8 @@ def consented_to_injection? = consented_vaccine_methods.include?("injection")
 
   def consented_vaccine_methods
     @consented_vaccine_methods ||=
-      patient.consent_status(programme:).vaccine_methods
+      draft_consent_vaccine_methods.presence ||
```

Comment:
This recent update was added to address a bug raised here: https://nhsd-jira.digital.nhs.uk/browse/MAV-1398.

If we don't pass in the `DraftConsent#vaccine_methods` when creating a new consent, we won't be able to display the correct label, i.e. "Yes, it’s safe to vaccinate (with injected vaccine only)".